src/encoding/text.rs

```diff
@@ -206,6 +207,88 @@ impl Encode for () {
     }
 }
 
+/// Warning: Using an IP address as a label is only useful when the number of
+/// distinct values is low (i.e. low cardinality). In all other cases you should
+/// combine your metrics into a single metric instead. Especially bad examples
```

Comment:
> In all other cases you should combine your metrics into a single metric instead.

I don't understand this. How is combining multiple metrics into one solving the issue of a cardinality explosion?

Reply:
Ah, as in dropping the label?